README.md

```diff
@@ -10,17 +10,31 @@ Test via ```nosetests```.
 ### Usage: 
     POST   on /shopcarts: creates new shopcart.
         Expects JSON body data:
-            item_id: -1
+            item_id: 0
```

Comment:
This is not right. This route has two uses, one is for 'creating an empty shopcart' given a user_id, which doesn't really do anything; one is for creating such a shopcart but also pass in either a list of items in the variable req.items in the implementation, or just one item, specified via item_id, name etc. If it's difficult to read the code and the exact implementation I or Yuwen who implemented this can write this part

Reply:
Understood. I have wrote a draft and committed again. I am not sure if this version fully reflects the meaning of your and Yuwen's code. If there's any problem, just let me know and I will fix it ASAP. Thanks Timothy!